build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        google()
```

Comment:
Google repo was already added

Reply:
Maybe it's conflict when I pulled request and merge with my branch.